src/main/resources/com/puppycrawl/tools/checkstyle/checks/modifier/messages_tr.properties

```diff
@@ -5,3 +5,4 @@ annotation.order = ''{0}'' anotasyon niteleyicisi, anotasyon olmayan niteleyicil
 mod.order = ''{0}'' niteleyicisi, Java tarafından önerilen sırada değil.
 
 redundantModifier = Gereksiz ''{0}'' niteleyicisi.
+interface.implied.modifier=Z\u0131mni de\u011Fi\u015Ftirici aç\u0131k olmal\u0131d\u0131r: ''{0}''
```

Comment:
@precoder, it would be awesome is you can suggest Turkish translation.

Reply:
After reading the discussion link at the top of issue, I would suggest something like:
interface.implied.modifier=Varsayılan niteleyici: ''{0}'' açıkça belirtilmelidir.